packages/gestalt/src/contexts/ScrollBoundaryContainer.js

```diff
@@ -34,6 +34,8 @@ function ScrollBoundaryContainerProvider({ children }: Props): Element<typeof Pr
   const scrollBoundaryContainerContext = {
     scrollBoundaryContainerRef,
     addRef: useCallback((ref) => {
+      // Cannot call `setScrollBoundaryContainerRef` with `ref` bound to the first parameter because a call signature declaring the expected parameter / return type is missing in  `HTMLElement` [1] but exists in  function type [2].
```

Comment:
I'm a bit confused by this one. Not sure where I should be adding a return type

Reply:
Would be good to figure this one out before merging.

Reply:
Does that error message suggest anything to you? I can't tell what it's trying to communicate.

Reply:
try 

` scrollBoundaryContainerRef: ?HTMLElement,`

Reply:
genius! 😅